package.json

```diff
@@ -13,24 +13,28 @@
     ],
     "scripts": {
         "prepare": "husky install",
-        "test": "jest --config jestconfig.json",
+        "test": "jest",
         "build": "tsc -p ./",
         "watch": "tsc -watch -p ./",
         "lint": "eslint --fix \"{src,tests}/**/*.{js,ts}\"",
         "lint-ci": "eslint \"{src,tests}/**/*.{js,ts}\"",
-        "test-coverage": "jest --config jestconfig.json --ci --reporters=jest-junit --reporters=default --coverage --coverageReporters=cobertura --coverageReporters=html",
+        "test-coverage": "jest --ci --reporters=jest-junit --reporters=default --coverage --coverageReporters=cobertura --coverageReporters=html",
         "generate-docs": "typedoc src/index.ts --options .docs-config/typedoc.json"
     },
     "keywords": [
         "Crowdin"
     ],
     "license": "MIT",
+    "dependencies": {
+        "axios": "^1"
```

Comment:
use a flex range to best support customers. all >=1.x supports the axios error class and compat interface